compat/os/opensolaris_10_netdb.h

```diff
@@ -74,15 +74,21 @@
  * End BIND 4.9.3
  */
 
+#ifndef _NETDB_H
+#define _NETDB_H
+
+/****************************************************************************
+ *--------------------------------------------------------------------------*
+ * DO *NOT* MAKE ANY CHANGES below here unless you know what you're doing...*
+ *--------------------------------------------------------------------------*
+ ****************************************************************************/
+
```

Comment:
As you know from another [thread](https://github.com/squid-cache/squid/pull/1341#discussion_r1183848944), I believe these warnings do more harm than good. I am against adding more of them. I will support a PR removing the existing ones.